lua/graph.lua

```diff
@@ -873,6 +873,11 @@ function filter_tags_generic(kv)
     return 1
   end
 
+  --toss actual areas
```

Comment:
this was way down at the bottom, if we are going to bail, best to do it early and save everyone some time :smile: